math/src/test/java/coppercore/math/GearConversionTest.java

```diff
@@ -0,0 +1,84 @@
+package coppercore.math;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class GearConversionTest {
+    private static final double delta = 1 * Math.pow(10.0, -10.0);
+
+    @Test
+    public void pitchDiameterFrom3mmPulleyTest() {
+        Assertions.assertEquals(
+                12 / (Math.PI), GearConversionFunctions.pitchDiameterFrom3mmPulley(4), delta);
+        Assertions.assertEquals(
+                150 / (Math.PI), GearConversionFunctions.pitchDiameterFrom3mmPulley(50), delta);
+        Assertions.assertEquals(
+                0 / (Math.PI), GearConversionFunctions.pitchDiameterFrom3mmPulley(0), delta);
+        Assertions.assertEquals(
+                21 / (Math.PI), GearConversionFunctions.pitchDiameterFrom3mmPulley(-7), delta);
```

Comment:
Let's make this (and other functions) a little cleaner by using arrays and a loop:
```
final double[] kInputs = {4.0, 50.0, 0.0, -7.0};
final double[] kExpectedOutputs = {12.0, 150.0, 0.0, 21.0};

for (int i = 0; i < kInputs.length; i++) {
    Assertions.assertEquals(kExpectedOutputs[i] / (Math.PI), GearConversionFunctions.pitchDiameterFrom3mmPulley(kInputs[i]), delta);
}
```

This way the test inputs and expected outputs are super clear, and the test is set up in a simple way that is easy to read